python/whylogs/core/feature_weights.py

```diff
@@ -0,0 +1,76 @@
+import json
+from typing import Any, Dict, Tuple, TypedDict
+
+from whylogs.api.writer import Writers
+from whylogs.api.writer.writer import Optional, Writable, Writer
+
+
+class FeatureWeightResponse(TypedDict):
+    segmentWeights: list
+    metadata: Dict
+
+
+class FeatureWeightDict(TypedDict):
+    segment: Optional[str]
```

Comment:
Can you make this an `Optional[Segment]`? The segment key is a Tuple of strings, so the single string will make this hard to support multi-column segments.

Reply:
so we're assuming that the songbird API will change? currently it's a string.

Reply:
Right, and if you get more than one string for the Segment key, then we'd have to figure out how to translate that to the single string value, but better to keep our API consistent in the whylogs side and do the translations right before upload if needed.

Reply:
Got it. So I'll make it an Optional[Segment], but raise an error if it's actually passing a Segment for now, since that wouldn't work with Songbird at the moment (and Segments is currently not supported by the platform)